assets/constants.ts

```diff
@@ -4769,9 +4769,6 @@ export default {
   }, {
     "code": "da",
     "description": "dansk (Danish)"
-  }, {
-    "code": "prs",
-    "description": "دری (Dari)"
```

Comment:
Earlier Dari was added to the content list though it was expected to be added for the translation and vocieover.